examples/shared_samples/ethernet_network.rb

```diff
@@ -37,7 +37,7 @@
 ethernet_class = OneviewSDK.resource_named('EthernetNetwork', @client.api_version)
 
 # Scope class used in this sample
-scope_class = OneviewSDK.resource_named('Scope', @client.api_version) unless @client.api_version.to_i <= 200
+scope_class = OneviewSDK.resource_named('Scope', 1200) unless @client.api_version.to_i <= 200
```

Comment:
please don't hardcode this apiversion here

Reply:
Reverted back